helm-charts/azure-api-management-gateway/templates/service.yaml

```diff
@@ -10,8 +10,13 @@ metadata:
   {{- end }}
 spec:
   type: {{ .Values.service.type }}
+{{- if eq .Values.service.type "LoadBalancer" }}
 {{- if (.Values.service.loadBalancer).ip }}
   loadBalancerIP: {{ .Values.service.loadBalancer.ip }}
+{{- end }}
+{{- if eq .Values.service.allocateLoadBalancerNodePorts false }}
```

Comment:
Can you add this to the values.yml file please?

Reply:
is it needed? 
as per kubernetes [documentation](https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/#ServiceSpec), allocateLoadBalancerNodePorts is defaulted to true

Reply:
It is best practice though, otherwise it's hard to track the knobs that we expose and planning to generate Helm chart docs based on it (https://github.com/Azure/api-management-self-hosted-gateway/issues/259)

Reply:
Thank you!